adapter/PreciceInterface.c

```diff
@@ -394,15 +396,15 @@ void PreciceInterface_ConfigureNodesMesh( PreciceInterface * interface, Simulati
 	//printf("numNodes = %d \n", interface->numNodes);
 	interface->nodeIDs = &sim->ialset[sim->istartset[interface->nodeSetID] - 1]; //Lucia: make a copy
 
-	interface->nodeCoordinates = malloc( interface->numNodes * 3 * sizeof( double ) );
-	getNodeCoordinates( interface->nodeIDs, interface->numNodes, sim->co, sim->vold, sim->mt, interface->nodeCoordinates );
+	interface->nodeCoordinates = malloc( interface->numNodes * interface->dim * sizeof( double ) );
+	getNodeCoordinates( interface->nodeIDs, interface->numNodes, interface->dim, sim->co, sim->vold, sim->mt, interface->nodeCoordinates );
 
 	if( interface->nodesMeshName != NULL )
 	{
 		//printf("nodesMeshName is not null \n");
 		interface->nodesMeshID = precicec_getMeshID( interface->nodesMeshName );
-		//interface->preciceNodeIDs = malloc( interface->numNodes * sizeof( int ) );
-		interface->preciceNodeIDs = malloc( interface->numNodes * 3 * sizeof( int ) );
```

Comment:
why was `preciceNodeIDs` x3?

Reply:
This originates from the nearest-projection mapping additions. Your change looks correct to my understanding and I would even remove the commented-out part if @KyleDavisSA agrees.